### PR TITLE
cli/bearer: Add a simple binary token reader

### PR DESCRIPTION
**Pull request by carpawell**
I wrote code for this operation more than twice. Naming/useful flags suggestions are welcome

### CODE REVIEW

File: cmd/neofs-cli/modules/bearer/pretty.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+var prettyCmd = &cobra.Command{
+	Use:   "pretty",
+	Short: "Print binary-marshalled bearer tokens from STDIN in JSON format",
+	Args:  cobra.NoArgs,
+	Run:   prettyToken,
+}
+
+func prettyToken(cmd *cobra.Command, _ []string) {
+	reader := cmd.InOrStdin()
```
[the remark this answers is not in the document]
Comment (carpawell) [reply cmd/neofs-cli/modules/bearer/pretty.go:19]: what do you think about `base58`, `base64` utils?

> base64 [OPTION]... [FILE]
> ...
> With no FILE, or when FILE is -, read standard input.

> base58 [-h] [-d] [-c] [FILE]
> ...
> Base58 encode or decode FILE, or standard input, to standard output.

can read arg and wait for path, but when missing or `-` read sdtin?

[the remark this answers is not in the document]
Comment (roman-khimov) [reply cmd/neofs-cli/modules/bearer/pretty.go:19]: That's the best thing to have, as usual.

File: cmd/neofs-cli/modules/bearer/pretty.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import (
+	"io"
+
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/common"
+	"github.com/nspcc-dev/neofs-sdk-go/bearer"
+	"github.com/spf13/cobra"
+)
+
+var prettyCmd = &cobra.Command{
+	Use:   "pretty",
```
[the remark this answers is not in the document]
Comment (carpawell) [reply cmd/neofs-cli/modules/bearer/pretty.go]: renamed

File: cmd/neofs-cli/modules/bearer/pretty.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (carpawell) [reply cmd/neofs-cli/modules/bearer/pretty.go]: done

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**